Makefile

```diff
@@ -27,16 +33,23 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 	pip install -qr requirements/pip-tools.txt
 	$(PIP_COMPILE) -o requirements/pip-tools.txt requirements/pip-tools.in
 	$(PIP_COMPILE) -o requirements/base.txt requirements/base.in
+	$(PIP_COMPILE) -o requirements/quality.txt requirements/quality.in
 	$(PIP_COMPILE) -o requirements/test.txt requirements/test.in
+	$(PIP_COMPILE) -o requirements/tox.txt requirements/tox.in
+
+python-quality-test: clean ## check coding style with pycodestyle and pylint
+	$(TOX) pycodestyle --config=setup.cfg ./openedx_external_enrollments
+	$(TOX) pylint ./openedx_external_enrollments --rcfile=./setup.cfg
+	$(TOX) isort --check-only --recursive --diff ./openedx_external_enrollments
 
-quality: clean ## check coding style with pycodestyle and pylint
-	pycodestyle ./openedx_external_enrollments
-	pylint ./openedx_external_enrollments --rcfile=./setup.cfg
-	isort --check-only --recursive --diff ./openedx_external_enrollments
+python-test: clean ## Run test suite.
+	#pip install -r requirements/test.txt --exists-action w
```

Comment:
Please uncomment this line. @ivanvgh

Reply:
Removed. Dependencies are being installed through 'install-dev-dependencies' command, in this case I am handling it like pearson-core plugin in order to reuse the GitHub actions workflow template.

Reply:
I think it was a bad idea to remove the installation of the requirements from each command because you should be able to run the commands standalone a not depend on other commands... Let's keep it as it is.